### PR TITLE
[ENG-2351][ENG-2353] Fix RWE registration form

### PR DESCRIPTION
**Pull request by brianjgeiger**
## Purpose

Fix problems with the RWE registration and the Qualitative registration forms

## Changes

1. Change 'description' blocks to 'help' blocks
2. Fix Q7 to match what the spreadsheet had
3. Add newlines to various places in the forms where they were called for
4. Update qualitative form based on feedback [ENG-2353]

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Examples show up below the help text
- Formatting is better on questions that have newlines

## Ticket

https://openscience.atlassian.net/browse/ENG-2351
https://openscience.atlassian.net/browse/ENG-2353

[ENG-2353]: https://openscience.atlassian.net/browse/ENG-2353

### CODE REVIEW

**Comment by UdayVarkhedkar** [comment]
Should the `description` blocks be changed to `help` blocks in the other newer schemas with `example` blocks as well for whenever they become active?

**Comment by brianjgeiger** [comment]
@UdayVarkhedkar Supposedly the other forms are good, so I don't _think_ we need to do that.

**Comment by brianjgeiger** [comment]
@UdayVarkhedkar Some changes for the Qualitative form have come through, I'm going to add them to this pr.

File: website/project/metadata/real-world-evidence.json [hunk 1]
```diff
@@ -89,25 +89,25 @@
                "type":"string",
                "format":"textarea",
                "title":"Study population",
-               "description":"Please choose one the population under evaluation in the study",
-               "example":"The population under evaluation in the study",
+               "help":"The population under evaluation in the study",
```
Comment (UdayVarkhedkar) [inline website/project/metadata/real-world-evidence.json:92]: The JIRA ticket makes it seem like the `help` value should be `Please choose one the population under evaluation in the study.` (maybe that should be `Please choose the population under evaluation in the study.`?)

**Review by UdayVarkhedkar — CHANGES_REQUESTED** [review]
Everything looks good except for the question I commented on q8 for the help text

**Review by UdayVarkhedkar — APPROVED** [review]

**Pull request merged**